python/spear/config/default_config.simulation_controller.yaml

```diff
@@ -110,6 +110,13 @@ SIMULATION_CONTROLLER:
     ACTION_COMPONENTS: ["control_joints"]
     OBSERVATION_COMPONENTS: ["link_state", "camera"]
     STEP_INFO_COMPONENTS: []
+    SPAWN_MODE: "player_start" # "player_start", "world_transform"
```

Comment:
Don't need to check this in because it has been cherry-picked into https://github.com/isl-org/spear/pull/176